services/nostr-band/nostr-band-followers.tester.js

```diff
@@ -0,0 +1,28 @@
+import Joi from 'joi'
+import { isMetric } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('fetch: valid npub')
+  .get('/npub18c556t7n8xa3df2q82rwxejfglw5przds7sqvefylzjh8tjne28qld0we7.json')
+  .expectBadge({
+    label: 'followers',
+    message: Joi.alternatives().try(isMetric, Joi.equal('invalid pubkey')),
```

Comment:
Can we make the expectation here just `isMetric`. If this example becomes invalid we'll update it.

Reply:
done: https://github.com/badges/shields/pull/9870/commits/6f75cffe2beef226c339afa3a99988278ad68aff